xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/Title.xml

```diff
@@ -53,6 +53,7 @@
   #end
   {{html clean="false"}}
   &lt;input type="text" name="$name" value="$!escapetool.xml($value)"
+    aria-label="$escapetool.xml($services.localization.render('platform.appwithinminutes.title.edit.label'))"
```

Comment:
Same here, I wonder if this is really needed. I've just checked and indeed the title label has a `for` attribute with an ID that looks good (e.g., `AI.Documents.Code.DocumentsClass_0_title` in the index for the LLM application) but the input doesn't have the expected ID. I would suggest setting an ID attribute that matches the generated label.

Reply:
When on the ClassEditSheet, the label of the field becomes itself editable. This means that what's usually a label now contains an input. This is implicit input labelling. I'm not sure HTML contains a definite way to handle such a case (label with both implicit and explicit input labelling) and I'd rather avoid meeting it. 

![image](https://github.com/user-attachments/assets/c88ff66b-d989-4b16-89fa-a14e92bdc762)


We could "duplicate" the label but I'd rather keep things as simple as possible and just have an `aria-label`.

Reply:
So there are two cases:

1. We're inside the AWM editor that uses `ClassEditSheet`.
2. We're inside the generated app.

I was primarily talking about the second case and didn't notice the first one. For the second case, I kind of agree, but to me this doesn't mean that we shouldn't consider the second case which should be the much more frequent case in which we have a visible label that doesn't contain another input that is not correctly connected with the input. For the first case, I'm wondering if we couldn't remove the `for` attribute on the existing label and add a `sr-only` second label that has the `for` attribute. In this first case, the input of the pretty name might also need an explicit `aria-label` as the content of the `label` element is wrong, this isn't the label but some hack for resizing the input.